javaparser-core-testing/src/test/java/com/github/javaparser/ast/expr/LambdaExprTest.java

```diff
@@ -27,15 +31,45 @@ private void assertRange(String startToken, String endToken, Node node) {
     }
 
     @Test
-    void getExpressionBody(){
+    void getExpressionBody() {
         LambdaExpr lambdaExpr = parseExpression("x -> y").asLambdaExpr();
         assertEquals("Optional[y]", lambdaExpr.getExpressionBody().toString());
     }
 
     @Test
-    void getNoExpressionBody(){
+    void getNoExpressionBody() {
         LambdaExpr lambdaExpr = parseExpression("x -> {y;}").asLambdaExpr();
         assertEquals("Optional.empty", lambdaExpr.getExpressionBody().toString());
     }
 
+    @Test
+    void oneParameterAndExpressionUtilityConstructor() {
+        LambdaExpr expr = new LambdaExpr(new Parameter(new UnknownType(), "a"), parseExpression("5"));
+        assertEquals("a -> 5", expr.toString());
+    }
+
+    @Test
+    void oneParameterAndStatementUtilityConstructor() {
+        LambdaExpr expr = new LambdaExpr(new Parameter(new UnknownType(), "a"), parseStatement("return 5;"));
+        assertEquals("a -> return 5;", expr.toString());
```

Comment:
I don't think this is valid Java code...

Reply:
Ooooooops!